src/tools/demo.ts

```diff
@@ -0,0 +1,57 @@
+export enum CommentType {
+  REMOVE_FILE = `@demo remove-file`,
+  REMOVE_CURRENT_LINE = `@demo remove-current-line`,
+  REMOVE_NEXT_LINE = `@demo remove-next-line`,
+  REMOVE_BLOCK_START = `@demo remove-block-start`,
+  REMOVE_BLOCK_END = `@demo remove-block-end`,
+}
+
+/**
+ * Take the file content as a string and remove any
+ * line of code with an `// @demo remove-current-line` comment
+ */
+function removeCurrentLine(contents: string, comment = CommentType.REMOVE_CURRENT_LINE): string {
+  const lines = contents.split("\n")
+  const result = lines.filter((line) => !line.includes(comment))
+  return result.join("\n")
+}
```

Comment:
I had a really nice experience with copilot writing these functions. I'd write the comment and the function name + types. Then I'd write the test. Then I'd go open up copilot suggestions and try different ones until it passed. I felt like I was just code reviewing my own code 😆